user-service/controllers/history.controller.js

```diff
@@ -6,28 +6,19 @@ const Op = db.Sequelize.Op;
 const sequelize = db.Sequelize;
 
 // Controller function to create a user history
-// Usage: Post request to http://localhost:3003/api/user/history
 // NEW Usage: Post request to http://localhost:3003/api/hist/save
```

Comment:
Think we can remove the `NEW` in the comment as this might confuse readers! 

```suggestion
// Usage: Post request to http://localhost:3003/api/hist/save
```